samples/drivers/led_eos_s3_1/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <soc.h>
+
+void main(void)
+{
+    	printk("\n\n");
+    	printk( "##########################\n");
+    	printk( "Quicklogic Open Platform 2.0\n");
+    	printk( "SW Version: ");
+    	printk("OP2-QuickFeather-led-eos-s3-sample1");
+    	printk( "\n" );
+    	printk( __DATE__ " " __TIME__ "\n" );
+    	printk( "##########################\n\n");
+
+	while (1);
```

Comment:
I think it is not needed since controller in FPGA will drive LED infinitely, is that correct? When Zephyr finishes `main` function it switches to `idle` mode, so there is no reason for busy wait.